vmaas-go/base/utils/config.go

```diff
@@ -143,6 +144,7 @@ func initEnv() {
 	Cfg.VmaasLibMaxGoroutines = GetIntEnvOrDefault("VMAAS_LIB_MAX_GOROUTINES", 20)
 	Cfg.NewerReleaseverRepos = GetBoolEnvOrDefault("NEWER_RELEASEVER_REPOS", true)
 	Cfg.NewerReleaseverCsaf = GetBoolEnvOrDefault("NEWER_RELEASEVER_CSAF", true)
+	Cfg.VmaasVersionFilePath = Getenv("VMAAS_VERSION_FILE_PATH", "")
```

Comment:
set the default as well so /version endpoint shows version while using docker-compose, or is it not necessary?